<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>coach_info_form</class>
 <widget class="QWidget" name="coach_info_form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>COACH INFO</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>500</width>
     <height>300</height>
    </rect>
   </property>
   <property name="title">
    <string>COACH INFO</string>
   </property>
   <widget class="QPushButton" name="create_coach_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>CREATE COACH</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>321</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>NAME:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_edit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>321</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>SURNAME:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="surname_edit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>321</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>NATIONALITY:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="nationalities_combo">
       <item>
        <property name="text">
         <string>Argentina</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Brazil</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Croatia</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Germany</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hungary</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>India</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Serbia</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Spain</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Turkey</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USA</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
